upload-sarif/action.yml

```
@@ -20,6 +20,7 @@ inputs:
     description: "The sha of the HEAD of the ref where results will be uploaded. If not provided, the Action will use the GITHUB_SHA environment variable. If provided, the ref input must be provided as well. This input is ignored for pull requests from forks."
     required: false
   token:
+    description: "Token for accessing /code-scanning/analysis endpoints (used to publish sarif -- generally requires `security-events: write`)"
```

Same as other comments!
```suggestion
    description: "GitHub token to use for authenticating with this instance of GitHub. The token needs the `security-events: write` permission."
    required: false
```